css-brochure-form/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Brochure Form</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Learn More</h1>
+    <div class="container">
+      <form>
+        <div class="input">
+          <label for="email">Company Email</label>
+          <br />
```

Comment:
This is valid, but I would encourage you not to use `<br />` for spacing between elements. We want to handle all styling in CSS. It would be better to make `label` a block level element to get it to appear on a new line, or `margin` to create space

Reply:
Thank you!